patroni/ha.py

```diff
@@ -257,9 +271,15 @@ def manual_failover_process_no_leader(self):
         return self._is_healthiest_node(members, check_replication_lag=False)
 
     def is_healthiest_node(self):
+        if self.is_paused() and self.cluster.failover and not self.cluster.failover.scheduled_at:
```

Comment:
> failover will lead to having more than one master under this logic

In theory it could happen, but in the real life somebody should "trigger" manual failover and manually promote one of the nodes nearly at the same time in order to get into such situation.